Externals/wxWidgets3/include/wx/features.h

```diff
@@ -73,15 +73,8 @@
     #define wxHAS_NATIVE_TAB_TRAVERSAL
 #endif
 
-/* This is defined when the compiler provides some type of extended locale
-   functions.  Otherwise, we implement them ourselves to only support the
-   'C' locale */
-#if defined(HAVE_LOCALE_T) || \
-    (wxCHECK_VISUALC_VERSION(8))
-    #define wxHAS_XLOCALE_SUPPORT
-#else
-    #undef wxHAS_XLOCALE_SUPPORT
-#endif
+/* This define is unused by dolphin. */
```

Comment:
A more proper way to do this would be to just set `wxUSE_XLOCALE` to `0` in [wx/wxcocoa.h](https://github.com/dolphin-emu/dolphin/blob/master/Externals/wxWidgets3/wx/wxcocoa.h#L252), [wx/wxgtk.h](https://github.com/dolphin-emu/dolphin/blob/master/Externals/wxWidgets3/wx/wxgtk.h#L242), and [wx/wxmsw.h](https://github.com/dolphin-emu/dolphin/blob/master/Externals/wxWidgets3/wx/wxmsw.h#L468), since those config files actually enable the functionality.